server/provider/networkProvider.go

```diff
@@ -464,6 +467,11 @@ func (provider *networkProvider) ComputeTransactionFeeForMoveBalance(tx *transac
 		gasLimit += extraGasLimitGuardedTx
 	}
 
+	isRelayedV3 := len(tx.RelayerAddress) > 0
```

Comment:
In the future, we might (should) remove the function `ComputeTransactionFeeForMoveBalance` from rosetta.

See MX-16423.

Reply:
perhaps extend this to check the len of relayerSignature as well?

Reply:
Done.